### DISCUSSION TITLE
Hoppscotsch question REST APIs

### DISCUSSION THREAD

**Discussion by IoanaCalugarescu** [hoppscotch/hoppscotch#4338] [General]
Hoppscotsch application can read automatically files and send then API calls to another application ?
Could you please provide more information regarding this matter?
Thank you for help.

**Comment by liyasthomas** [comment]
Here are the links to our documentation to learn more about:
- [APIs for User Management](https://docs.hoppscotch.io/documentation/self-host/enterprise-edition/admin-dashboard#apis-for-user-management)
- [APIs for Workspace Management](https://docs.hoppscotch.io/documentation/self-host/enterprise-edition/admin-dashboard#apis-for-workspace-management)
- [Infra tokens](https://docs.hoppscotch.io/documentation/self-host/enterprise-edition/admin-dashboard#infra-tokens)
